### ISSUE TITLE
parse failure when using parentheses-less function application in an else/else if/unless block

### ISSUE THREAD

**Issue by WuTheFWasThat** [decaffeinate/decaffeinate#317]
```
if true
  console.log 'hello'
else
  console.log 'fails'
```

fails with
```
# decaffeinate test.coffee
test.coffee → test.js
test.coffee: AddVariableDeclarationsStage failed to parse: Unexpected token (4:14)
  2 |   console.log('hello');
  3 | } else {
> 4 |   console.log 'fails';
    |               ^
  5 | }
  6 |
```

**Comment by alangpierce** [comment]
Seems to be working for me in the repl:
http://decaffeinate-project.org/repl/#?evaluate=true&code=if%20true%0A%20%20console.log%20'hello'%0Aelse%0A%20%20console.log%20'fails'

I fixed this recently with https://github.com/decaffeinate/decaffeinate/commit/59fca2400ef97e05c32caffa191fad825cd8042a , so make sure you're on the latest version.

**Comment by eventualbuddha** [comment]
Thanks! @alangpierce is correct that this is fixed as of [v2.17.1](https://github.com/decaffeinate/decaffeinate/releases/tag/v2.17.1).
